tensorboard/plugins/interactive_inference/utils/platform_utils.py

```diff
@@ -14,9 +14,11 @@
 # ==============================================================================
 """Shared utils among inference plugins that are platform-specific."""
 
+import csv
 from glob import glob
 from grpc.beta import implementations
 import random
+import numpy as np
```

Comment:
is np used?

Reply:
good catch. not anymore. removed from here and build file